catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -330,6 +337,10 @@ public void setAdditionalAttributes(List<String> attributes) {
         additionalMetacardAttributes = Permissions.parsePermissionsFromString(attributes);
     }
 
+    public void setBodyExpansion(String bodyExpansion) {
+        this.bodyExpansion = bodyExpansion;
```

Comment:
❓ Should we do any kind of validation here? Would it be possible for someone to do something malicious with this?